prog/minio/setup_minio.rb

```diff
@@ -26,6 +26,7 @@ class Prog::Minio::SetupMinio < Prog::Base
 MINIO_ROOT_USER="#{minio_server.cluster.admin_user}"
 MINIO_ROOT_PASSWORD="#{minio_server.cluster.admin_password}"
 #{server_url_config}
+MINIO_STORAGE_CLASS_STANDARD="EC:#{minio_server.pool.per_server_drive_count * minio_server.pool.servers.count / 2}"
```

Comment:
I don't thinks this will work for larger number of disks or servers.

The formula I am looking at it this ([source](https://docs.min.io/community/minio-object-store/operations/concepts/erasure-coding.html#erasure-coding-basics)):

```
N (ERASURE SET SIZE) = K (DATA) + M (PARITY)
```

The parity value cannot exceed the Erasure set size, which can be 16 at max: https://docs.min.io/community/minio-object-store/operations/concepts/availability-and-resiliency.html#distributed-minio-deployments

And max parity can be half of the drives in an erasure set, so it cannot exceed 8. ([calculator](https://www.min.io/product/erasure-code-calculator)). I think the formula might be:

```suggestion
MINIO_STORAGE_CLASS_STANDARD="EC:#{[minio_server.pool.per_server_drive_count * minio_server.pool.servers.count / 2, 8].min}"
```